Sources_v3/WebSocketClient/WebSocketReconnectionStrategy.swift

```diff
@@ -19,23 +19,31 @@ class DefaultReconnectionStrategy: WebSocketClientReconnectionStrategy {
     static let maximumReconnectionDelay: TimeInterval = 25
     
     private var consecutiveFailures = 0
+    private let inernetConnection: InternetConnection
+    
+    init(inernetConnection: InternetConnection) {
+        self.inernetConnection = inernetConnection
+    }
```

Comment:
🤷‍♂️

Reply:
We discussed that. `DefaultReconnectionStrategy` shouldn't retry reconnect while Internet is unavailable.

Reply:
I thought you'd something like this https://stackoverflow.com/a/31652395/3789847